lms/djangoapps/grades/api/urls.py

```diff
@@ -11,8 +11,14 @@
     '',
     url(
         r'^v0/course_grade/{course_id}/users/$'.format(
-            course_id=settings.COURSE_ID_PATTERN
+            course_id=settings.COURSE_ID_PATTERN,
         ),
         views.UserGradeView.as_view(), name='user_grade_detail'
     ),
+    url(
+        r'^v0/policy/{course_id}/$'.format(
+            course_id=settings.COURSE_ID_PATTERN,
+        ),
+        views.CourseGradingPolicy.as_view(), name='course_grading_policy'
```

Comment:
I think a more RESTful way to name this endpoint could be `v0/courses/{course_id}/policy`, though the full URI of `/api/grades/v0/courses/{course_id}/policy` does read a bit strangely. I think ideally the order would be something like `/api/v0/courses/{course_id}/policy` though I know the current grading API precludes us from using that. That would be more something to consider for the future.